buildconfig/CMake/CommonSetup.cmake

```diff
@@ -283,8 +283,9 @@ if ( CMAKE_VERSION VERSION_GREATER "3.5" )
     endif()
   else()
     set(CMAKE_CXX_CLANG_TIDY "" CACHE STRING "" FORCE) # delete it
-    message(AUTHOR_WARNING "Using cmake version 3.5 or below. Clang-tidy is not supported!")
+    message(AUTHOR_WARNING "ENABLE_CLANG_TIDY is off. Clang-tidy will not run!")
```

Comment:
Do we need this warning? I assume if someone does not set `ENABLE_CLANG_FORMAT` they wouldn't want it to run.

Reply:
Good point. I don't want to spam every dev with warnings when they probably haven't ever used clang-tidy